src/OrbitGl/FrameTrack.cpp

```diff
@@ -245,9 +244,10 @@ void FrameTrack::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset
   Vec2 white_text_box_position(pos_[0] + layout.GetRightMargin(),
                                y - layout.GetTextBoxHeight() / 2.f);
 
-  batcher->AddLine(from, from + Vec2(layout.GetRightMargin() / 2.f, 0), text_z, kWhiteColor);
-  batcher->AddLine(Vec2(white_text_box_position[0] + white_text_box_size[0], y), to, text_z,
-                   kWhiteColor);
+  Batcher* ui_batcher = canvas->GetBatcher();
```

Comment:
The batcher of the FrameTrack is changing here

Reply:
Was this a bug before?

Reply:
I believe that, but I'm not sure and added @ronaldfw because I wanted his opinion. 

Is the only draw function which used the timegraph batcher, and this batcher is only cleaned when we make UpdatePrimitives, so I think we were drawing these lines several times.

But this couldn't make any real performance implication neither causing a crash.